ompi/mpi/c/profile/Makefile.am

```diff
@@ -0,0 +1,496 @@
+# -*- makefile.am -*-
```

Comment:
Do you know what is going on here? It shows an empty file as the original but I see there is a non-empty file here.

Reply:
removed.  a mistake in a rebase.